tests/gtest/avifdecodetest.cc

```diff
@@ -29,6 +29,26 @@ TEST(AvifDecodeTest, ColorGridAlphaNoGrid) {
   EXPECT_GT(decoder->image->alphaRowBytes, 0u);
 }
 
+TEST(AvifDecodeTest, AnimatedImage) {
+  if (!testutil::Av1DecoderAvailable()) {
+    GTEST_SKIP() << "AV1 Codec unavailable, skip test.";
+  }
```

Comment:
I guess this could be moved right before the first call to `avifDecoderNextImage()` to exercise `avifDecoderParse()` when there is no AV1 codec, but at the same time it breaks the test logic flow.

Reply:
Ack, while that's theoretically true, there's not much point in testing `avifDecoderParse` when no AV1 decoder is available.